### ISSUE TITLE
Discord notification: Link to movie page on radarr

### ISSUE THREAD

**Issue by d3mystified** [Radarr/Radarr#7964]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

When I get a discord notification, it doesn't link to the radarr page for that movie.

### Describe the solution you'd like

It would be nice to link to the movie, maybe more prominently than other links are (IMDb etc).

### Describe alternatives you've considered

Manually searching in radarr 

### Anything else?

Nope

[AB#4638](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/4638)

**Comment by austinwbest** [comment]
Quick search for `local link` found
https://github.com/Radarr/Radarr/issues/7229

Which also points to
https://github.com/Radarr/Radarr/issues/5757

**Comment by austinwbest** [comment]
Guess it doesn't if you say it is completely different.

**Comment by bakerboy448** [comment]
Use the `Application URL` to be included in the Discord Notification and allow a link to view the movie page in Radarr?

**Comment by Qstick** [comment]
Not quite, there's a bit of logic that has to go into this to ensure it's correct. But it's a valid request and we'll keep it on the list
